app/reducers/payment.js

```diff
@@ -42,10 +43,13 @@ export function getPayments() {
 }
 
 export function sendPayment(data) {
-  const payment = Object.assign({}, data, {
+  const invoice = decodePayReq(data.paymentRequest)
+  const payment = {
+    ...data,
     status: 'sending',
-    timestamp: Math.round(new Date() / 1000),
-  })
+    creation_date: Math.round(new Date() / 1000),
+    payment_hash: invoice.tags.find(t => t.tagName === 'payment_hash').data,
```

Comment:
Probably worth adding a check for `invoice.tags`, but more importantly for the find result (just for some odd case when tag is not found)